webapp/platform/types/package.json

```diff
@@ -13,7 +13,11 @@
   "exports": {
     "./*": "./lib/*.js"
   },
-  "types": "./lib/*.d.ts",
+  "typesVersions": {
```

Comment:
I didn't know about this field, but it behaves similarly to `types` except it lets you do things that TS didn't previously support like exposing multiple files from a TS module. It's needed here because we import from things like `@mattermost/types/users` instead of importing everything from a single file like we do with the Client and Components packages